src/binaryen.cpp

```diff
@@ -492,6 +492,7 @@ ExecutionResult BinaryenEngine::execute(
 
   // NOTE: DO NOT use the optimiser here, it will conflict with metering
 
+  module.memory.initial = 20;
```

Comment:
Why is this needed?

Reply:
Runevm's deployer ran out of memory. This was a temporary work-around. Should chisel's deployer maybe run `grow_memory`?

Reply:
Right, it is due to this: https://github.com/wasmx/wasm-chisel/issues/53

Reply:
Removed this line.